atomic_reactor/plugins/pre_add_help.py

```diff
@@ -0,0 +1,81 @@
+"""
+Copyright (c) 2017 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+
+
+Convert a help markdown file a man page and store it to /help.1 in the image
+so that 'atomic help' could display it.
+This is accomplished by appending an ADD command to it.
+
+Example configuration:
+{
+    'name': 'add_help',
+    'args': {'help_file': 'help.md'}
+}
+"""
+
+import os
+from subprocess import check_output, CalledProcessError, STDOUT
+from atomic_reactor.plugin import PreBuildPlugin
+from atomic_reactor.util import df_parser
+
+
+class AddHelpPlugin(PreBuildPlugin):
+    key = "add_help"
+    man_filename = "help.1"
+    go_md2man_cmd = "go-md2man -in=$HELP_MD -out=$HELP_1"
```

Comment:
Can we explicitly double-quote the environment variables to make it safe?
`'go-md2man -in="$HELP_MD" -out="$HELP_1"'`